thirdparty/EVPath/EVPath/cm.c

```diff
@@ -358,7 +359,7 @@ INT_CMget_contact_list(CManager cm)
 extern attr_list
 INT_CMget_specific_contact_list(CManager cm, attr_list attrs)
 {
-    char *chosen_transport = NULL, *chosen_net = NULL;
+    char *chosen_transport = NULL, *chosen_net = NULL, *chosen_interface;
```

Comment:
You forgot to initialize chosen_interface with NULL. it causes a segfault in line 411.

Reply:
Yeah, I was just diagnosing that and another issue.  CI is really slow.  Pushing something else soon...